app/src/main/java/zapsolutions/zap/lnurl/Bech32.java

```diff
@@ -0,0 +1,152 @@
+package zapsolutions.zap.lnurl;
+
+import java.util.Locale;
+
+public class Bech32 {
```

Comment:
Did you write this decoding logic yourself? 👀

Reply:
No, I the Bech32 Class was written by SamouraiDev and I found it here:
https://github.com/sipa/bech32/pull/19/commits/b2c3b029f4dc8586bb1d01fcc2587de8efdfb5a0

Should I add a notice as comment in the code?